lib/rubocop/extension/generator/generator.rb

```diff
@@ -101,7 +101,7 @@ def self.defaults!
 
           patch "#{name}.gemspec", /^end/, <<~RUBY
 
-              spec.add_runtime_dependency 'rubocop'
+              spec.add_runtime_dependency 'rubocop', '>= 1.22.0'
```

Comment:
I think the `gemspec` generated by the generator  should be have this too.

Reply:
Hm. I think it's better not to have this version lock because the dependency lock context of the rubocop-extension-generator gem and a generated 3rd party cop gem are different.

Reply:
I see. I removed this.